domain-model-assistant/Assets/Components/Scripts/Diagram.cs

```diff
@@ -203,26 +167,39 @@ public void LoadJson(string cdmJson)
     {
         ResetDiagram();
         //Debug.Log(cdmJson);
-        var classDiagram = JsonUtility.FromJson<ClassDiagramDTO>(cdmJson);
+        var cdmDto = JsonUtility.FromJson<ClassDiagramDTO>(cdmJson);
 
         //store attributes of class in a dictionary
-        classDiagram.classes.ForEach(cls => this.classIdToAttributes[cls._id] = cls.attributes);
+        cdmDto.classDiagram.classes.ForEach(cls => classIdToAttributes[cls._id] = cls.attributes);
         //store attribute types. Map type id to eclass tye
-        classDiagram.types.ForEach(type =>
+        cdmDto.classDiagram.types.ForEach(type =>
         {
             //cache eClass attr with shortened substring
             //Eg. http://cs.mcgill.ca/sel/cdm/1.0#//CDString -> string
-            string res = type.eClass.Substring(type.eClass.LastIndexOf('/') + 1).Replace("CD", "").ToLower();
+            string res = type.eClass[(type.eClass.LastIndexOf('/') + 1)..].Replace("CD", "").ToLower();
```

Comment:
It is recommended to access substrings in C# using `[]` instead of `Substring()`